### PR TITLE
[raftstore-v2]: optimize the load based split config based on region size

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### What is changed and how it works?

Issue Number: Ref #12842

What's Changed:

```commit-message
1) optimize the load based split config based on region size
2) polish a log message when it cannot find a target peer of the message. 
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

### Release note 

```release-note
The default value of load based split feature is changed when the region size exceeds 4GB.
It will be 7000 and 0.75 for parameters split.qps-threshold and split.region-cpu-overload-threshold-ratio respectively. 
```

### CODE REVIEW

**Review by SpadeA-Tang — APPROVED** [review]

File: components/raftstore/src/store/worker/split_config.rs [hunk 1]
```diff
@@ -6,12 +6,16 @@ use lazy_static::lazy_static;
...
+    info,
+};
 
 const DEFAULT_DETECT_TIMES: u64 = 10;
 const DEFAULT_SAMPLE_THRESHOLD: u64 = 100;
 pub(crate) const DEFAULT_SAMPLE_NUM: usize = 20;
-const DEFAULT_QPS_THRESHOLD: usize = 3000;
+pub const DEFAULT_QPS_THRESHOLD: usize = 3000;
+pub const DEFAULT_BIG_REGION_QPS_THRESHOLD: usize = 7000;
 const DEFAULT_BYTE_THRESHOLD: usize = 30 * 1024 * 1024;
```
Comment (dbsid) [inline components/raftstore/src/store/worker/split_config.rs:19]: we should increase the bytes-threshhold as well, for example, in the stability test, we disabled load based split by increase the byte-threshhold

`SET config tikv split.byte-threshold=1572864000;`

**Review by dbsid — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Review by BusyJay — APPROVED** [review]

**Pull request merged**